### PR TITLE
Pathogenesis - Some responses to reviewer 2

### PR DESCRIPTION
**Pull request by RLordan**
Hey @rando2 - I added a bit on the variants. Also ticked off a few other small requests for reviewer 2. There are still some remaining as highlighted below.

Complete - Reviewer 2 points: 1,2,4,6,10, and 12

Points remaining - 3,5,7,8,9,11

### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/07.pathogenesis.md
Comment (RLordan) [inline content/07.pathogenesis.md:248]:
```suggestion
This finding suggests that IFN induction is dependent on the magnitude of exposure.
```

Comment (RLordan) [inline content/07.pathogenesis.md:248]:
```suggestion
This finding suggests that IFN induction is dependent on the extent of exposure.
```

Comment (RLordan) [reply content/07.pathogenesis.md:248]: My bad, deleted wrong one.

File: content/07.pathogenesis.md
Comment (RLordan) [inline content/07.pathogenesis.md:479]:
```suggestion
The extent to which aerosol and fomite transmission contribute to the spread of SARS-CoV-2 remains a question: in general, much like SARS-CoV-1 and MERS-CoV, this virus seems to be spread primarily by droplet transmission.
```

File: content/07.pathogenesis.md
Comment (RLordan) [inline content/07.pathogenesis.md:448]: This is just a suggestion rather than making an entirely new section, also shorten as you see fit

File: content/07.pathogenesis.md
Comment (RLordan) [inline content/07.pathogenesis.md:459]: Reference to evolution manuscript required.

**Comment by agitter** [comment]
I created a checklist in #892 that we can use to track which requests have been resolved by this pull request once it is merged.

**Comment by RLordan** [comment]
> I created a checklist in #892 that we can use to track which requests have been resolved by this pull request once it is merged.

Great idea, thanks @agitter

**Comment by rando2** [comment]
I'm so sorry for the slow response on this! I am at least partly back from the other project now and am hoping to get to this PR today (currently working on some of the vaccine PRs so I can help keep that moving forward!)

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:76]:
```suggestion
While, like most viruses, it is possible that SARS-CoV-1 and SARS-CoV-2 can enter cells through endocytosis,  a process conserved among coronaviruses enables them to target cells for entry through fusion with the plasma membrane [@doi:10.1038/cr.2008.15; @doi:10.1146/annurev-biochem-060208-104626].
```
I was thinking this might be what the reviewer meant, but yours could also be! I just figured I'd put it here to compare our thoughts!

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:460]:
```suggestion
For example, coordinated sequencing efforts of SARS-CoV-2 genomes in the U.K. were instrumental in detecting B.1.1.7/VOC 202012/01, a variant of concern (VOC) that, with an estimated R~0~ of 1.4, is associated with approximately 50% increased transmission due to genetic mutations in the _Spike_ gene that modify the Spike protein [@url:https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/955239/NERVTAG_paper_on_variant_of_concern__VOC__B.1.1.7.pdf; @doi:10.1101/2020.12.24.20248822; @doi:10.1016/j.cell.2021.01.044].<!--TO DO: What is the molecular rationale for why this is more virulent?-->
```
I think we mention the UK sequencing efforts in the evolution paper but not here, so I'm suggesting modifying the start of this sentence a bit -- but I could be mistaken about that.
Adding a comment that it would be good to try to take this to the next level pathogenesis-wise, but I don't want to hold up your PR any longer!

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:461]:
```suggestion
This variant has now spread to at least 93 other countries [@url:https://cov-lineages.org/global_report_B.1.1.7.html], and other studies investigating its transmission have occurred.
```

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:462]:
```suggestion
In the U.S. between December 2020 and January 2021, it was estimated that B.1.1.7 had an increased transmission of 35-45% relative to common SARS-CoV-2 variants at the time, and B.1.1.7 is now on its way to becoming the dominant SARS-CoV-2 variant in the U.S. [@doi:10.1101/2021.02.06.21251159].
```

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:463]:
```suggestion
Notably, the latest analyses in the U.K. reports that there may also be an increased risk of disease severity associated with B.1.1.7 [@url:https://www.gov.uk/government/publications/nervtag-paper-on-covid-19-variant-of-concern-b117] and potentially a 35% increased risk of death [@doi:10.1101/2021.02.01.21250959] compared to other common variants of SARS-CoV-2. 
```

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:465]:
```suggestion
Other variants meriting further investigation have also been identified worldwide, including the P.1 lineage associated with outbreaks in Brazil and the B.1.351 variant first identified in South Africa.
```
Not sure if my phrasing of not-VOC-but-we're-worried makes sense!

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:468]:
```suggestion
Indeed, these novel variants are being investigated for potential resistance to vaccines and antibody treatments such as Bamlanivimab, but to date it seems that these agents are still effective, albeit with reduced efficacy in the case of the Pfizer-BioNTech vaccine [@doi:10.1101/2021.02.02.428884]. 
```

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:469]:
```suggestion
As a consequence of reliance on targeting the SARS-CoV-2 Spike protein for many therapeutic and prophylactic strategies, increased surveillance is required to rapidly identify and prevent the spread of novel SARS-CoV-2 variants with alterations to the Spike protein.
```

**Review by rando2 — APPROVED** [review]
This reviewer is so helpful! I think these are great responses @RLordan and sorry I've been slow to come back to helping with this. I had one thing that I read a little differently than you did but I can definitely understand your reading as well, so not sure which they meant!

File: content/07.pathogenesis.md
Comment (rando2) [reply content/07.pathogenesis.md:459]: @agitter this seems great to me. I looked through that one today and I'm optimistic we can get it on arXiv soon-ish, although hard to tell exactly how things will line up with resubmission here.

File: content/07.pathogenesis.md
Comment (RLordan) [reply content/07.pathogenesis.md:76]: As I read back their comment, they say previous sentence, so I think you are correct.

File: content/07.pathogenesis.md
Comment (RLordan) [reply content/07.pathogenesis.md:460]: I think you are right. I agree re pathogenesis, a lot of new papers out. @agitter set up an issue this week for one that we may want to consider citing here, but that we should definitely cite in the evolution paper.

File: content/07.pathogenesis.md
Comment (RLordan) [reply content/07.pathogenesis.md:465]: Haha it does, at this stage they are all colloquially VOCs

File: content/07.pathogenesis.md
Comment (rando2) [inline content/07.pathogenesis.md:179]:
```suggestion
However, not all cases of severe COVID-19 in children are characterizable as MIS-C.
A recent study [@doi:10.1001/jama.2021.2091] described demographic and clinical variables associated with MIS-C in comparison with non-MIS-C severe acute COVID-19 in young people in the United States.

```

**Review by rando2 — COMMENTED** [review]
This is just to resolve the merge conflict with #885 (I don't want to create more mess since Ronan and I are editing this file at the same time)

**Comment by RLordan** [comment]
> This reviewer is so helpful! I think these are great responses @RLordan and sorry I've been slow to come back to helping with this. I had one thing that I read a little differently than you did but I can definitely understand your reading as well, so not sure which they meant!

Hey @rando2 , no problem, I think you were right on that line. Let me know if you want me to approve or commit anything else here. 

In terms of priority at the moment, Do you want me to focus on getting vaccines or evolution (update on variants) updated first. 

I also mentioned Amruta might be able to help with the vaccines manuscript. I met her today and she said she can give a brief synopsis on India's candidates. She just came back from India so she has all the vaccine gossip.

**Comment by rando2** [comment]
Hey @RLordan, that sounds awesome! From my own personal perspective (my classroom time is about to get much, much heavier as of April 5, and then I will probably have some other limitations come up right as the class ends), I think I'd prefer to focus on getting all of the preprints up and then tackle the gentler reviews, since that requires less creative energy (for me at least!)

I have a student coming on to help address Review 1.3-- he is going to read those 4 papers and then write up a synopsis (I'll tag him when he sets up his GitHub username). It would be great to get Amruta involved ASAP since I think vaccines is *so close*! I'm planning to sit down and engage with the pharma reviews today and then take a critical look at that one and vaccines this week.

**Comment by RLordan** [comment]
> Hey @RLordan, that sounds awesome! From my own personal perspective (my classroom time is about to get much, much heavier as of April 5, and then I will probably have some other limitations come up right as the class ends), I think I'd prefer to focus on getting all of the preprints up and then tackle the gentler reviews, since that requires less creative energy (for me at least!)
> 
> I have a student coming on to help address Review 1.3-- he is going to read those 4 papers and then write up a synopsis (I'll tag him when he sets up his GitHub username). It would be great to get Amruta involved ASAP since I think vaccines is _so close_! I'm planning to sit down and engage with the pharma reviews today and then take a critical look at that one and vaccines this week.

Gotcha! look forward to welcoming your student. I will try get Amruta involved ASAP. I will try fill in the blanks in the vaccine paper during the week.

**Pull request merged**
